dist/commit.hbs

```diff
@@ -0,0 +1,56 @@
+* {{header}}
```

Comment:
non-blocking
I am confused re: how the handlebars templates are used in the packaging

Reply:
Everything in `dist` is an output of the bundling.  I'm guessing that these were referenced in the `node_modules` and needed to be output to the `dist` directory to be referenced by the bundled version of the action.